package.json

```diff
@@ -106,7 +106,8 @@
     "validator": "^13.7.0",
     "vm-browserify": "^1.1.2",
     "websocket.js": "^0.1.7",
-    "zxcvbn": "^4.4.2"
+    "zxcvbn": "^4.4.2",
+    "mazemap": "file:./vendor/mazemap"
```

Comment:
I agree on the choice of a package.json `file:./vender` for this plugin. I looked at your `[WIP]` and was going to suggest you do it this way! Nice that you figured it out 🥇